core/integration/module_cli_test.go

```diff
@@ -676,6 +677,27 @@ func (CLISuite) TestDaggerInstall(ctx context.Context, t *testctx.T) {
 		})
 	})
 
+	t.Run("installing a dependency with duplicate name is not allowed", func(ctx context.Context, t *testctx.T) {
```

Comment:
Make sure to add a test for two dependencies without explicit `--name`s.

(though, note, this will probably fail until https://github.com/dagger/dagger/pull/9052 is merged - since at the point of checking, the `name` is actually empty :cry:)

Reply:
do we want to move the fix for this change to the other PR to keep similar functionality in same PR?